Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -816,7 +816,10 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
             throw error
         }
         
-        for analyzedDocument in analyzedDocuments {
+        // to preserve the order of documents by url
+        let analyzedDocumentsSorted = analyzedDocuments.sorted(by: {String($0.0.absoluteURL) > String($1.0.absoluteURL)})
```

Comment:
nit: This can use `absoluteString` or `path` instead.

Reply:
We have a nice KeyPath-based extension for this actually:

```suggestion
        let analyzedDocumentsSorted = analyzedDocuments.sorted(by: \.0.absolutePath)
```

Reply:
Thank you @d-ronnqvist and @ethan-kusters thats useful.

Reply:
@swift-ci please test